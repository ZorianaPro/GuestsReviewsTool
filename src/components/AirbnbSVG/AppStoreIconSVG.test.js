import React from 'react';
import { shallow } from 'enzyme';
import AppStoreIconSVG from './';

describe('AppStoreIconSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <AppStoreIconSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<AppStoreIconSVG />)).toMatchSnapshot();
  });
});
