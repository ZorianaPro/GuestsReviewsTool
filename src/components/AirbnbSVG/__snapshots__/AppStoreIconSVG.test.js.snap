// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppStoreIconSVG renders as expected 1`] = `
<svg
  className="AirbnbSVG"
>
  <use
    xlinkHref="#AirbnbSymbol"
  />
</svg>
`;
